configs/procgen/bigfish.ron

```diff
@@ -0,0 +1,29 @@
+ExperimentConfig(
+    env: (
+        id: "BigFish",
+    ),
+    rollout: (
+        num_envs: 256,
+        steps: 64,
+        processes: 16,
+    ),
+    total_timesteps: 10000000,
+    net: (
+        d_model: 64,
+        n_layer: 2,
+        translation: (
+            position_features: ["x", "y"],
+            reference_entity: "Player",
+            add_dist_feature: true,
+        ),
+    ),
+    optim: (
+        bs: 8192,
+        lr: 0.01,
+    ),
+    ppo: (
+        ent_coef: 0.1,
+        anneal_entropy: true,
+        gamma: 0.99,
+    ),
+)
```

Comment:
These are the hyperparameters from the procgen paper:

```
ExperimentConfig(
    env: (
        id: "BigFish",
    ),
    rollout: (
        num_envs: 64,
        steps: 256,
        processes: 1, // we should use process = 1 becasue procgen under the hood leverages multi threads and processes?
    ),
    total_timesteps: 25000000,
    net: (
        d_model: 64,
        n_layer: 2,
        translation: (
            position_features: ["x", "y"],
            reference_entity: "Player",
            add_dist_feature: true,
        ),
    ),
    optim: (
        bs: 8192,
        lr: 0.0005,
        max_grad_norm: 0.5,
    ),
    ppo: (
        ent_coef: 0.01,
        anneal_entropy: false,
        gamma: 0.999,
    ),
) 
``` 

derived from here: https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo_procgen.py, which results in very accurate reproduction:

![image](https://user-images.githubusercontent.com/5555347/159299400-35ae2413-9062-4d2e-b071-56f1aad7626a.png)

I'd suggest at least matching 

```
    num_envs: 64,
    steps: 256,
    total_timesteps: 25000000,
```